### PR TITLE
Feature/deployment fixes

### PR DESCRIPTION
**Pull request by combinatorist**
@Rian501, can you see if you are able to deploy with these additional instructions? I've still only gotten this to work on my Mac, so it might not work on your PC, but I think it's worth a try. If it still doesn't work for you, then I'll try my linux and then we can try yours together.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,8 +12,9 @@
     "lint:fix": "prettier --write 'src/**/*.{js,json,jsx}'"
   },
   "dependencies": {
+    "ajv": "^6.10.0",
     "firebase": "^5.11.1",
-    "firebase-admin": "~6.0.0",
+    "firebase-admin": "^7.4.0",
```
Comment (CoreyAR) [inline package.json]: Seems like we should also have a new lock file.

File: functions/package.json [hunk 2]
```diff
@@ -10,8 +10,11 @@
   },
   "dependencies": {
     "@sendgrid/mail": "^6.3.1",
-    "firebase-admin": "~6.0.0",
-    "firebase-functions": "^2.1.0"
+    "firebase-admin": "^8.2.0",
```
Comment (CoreyAR) [inline functions/package.json]: Seems like we should also have a new lock file.

File: package.json (hunk 1 above)
Comment (combinatorist) [reply package.json]: Yup - it didn't update because someone(...?) put the lock file in the .gitignore. ;)

Comment (combinatorist) [reply package.json]: Should I remove it from .gitignore?

Comment (CoreyAR) [reply package.json]: To avoid confusion the lock file from npm was ignored in favor or `yarn.lock`.

Comment (CoreyAR) [reply package.json]: Only one should be committed. Either from yarn or npm.

Comment (CoreyAR) [reply package.json]: We are using yarn in CI so we should probably update the `yarn.lock` unless there is a plan to switch CI to use npm.

Comment (combinatorist) [reply package.json]: Ok, does that mean I should do `yarn install` before each build? I'm trying to figure out how to add this to explanations in the README.

Comment (CoreyAR) [reply package.json]: No, the `yarn.lock` only changes when you change the dependencies.

Comment (CoreyAR) [reply package.json]: If it is confusing you might add that dependencies are managed using yarn to the README.md. This should be evident by the `yarn.lock` and yarn commands already in the README.md but it won't hurt to be more explicit.

**Comment by combinatorist** [comment]
Ok, I've got this working with yarn, but still only Mac. When I try on my linux machine, I get:
```
info fsevents@1.2.4: The platform "linux" is incompatible with this module.
info "fsevents@1.2.4" is an optional dependency and failed compatibility check. Excluding it from installation.
``` :(

**Pull request merged**